### PR TITLE
cmake: Add standard cmake build dir to gitignore

### PR DESCRIPTION
**Pull request by RobLoach**
The standard practice for CMake is to use a build directory...

```
mkdir build
cd build
cmake ..
make
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 *.creator*
 *.files
 *.includes
+build
```
Comment (gek169) [inline .gitignore:5]: Approved

**Review by gek169 — COMMENTED** [review]
looks good

**Comment by gek169** [comment]
thanks again for the contribution.
The cmakefiles.txt issue has been resolved, pardon the discrepancy

**Comment by RobLoach** [comment]
Thanks a lot! :smile:

**Comment by gek169** [comment]
I invited you to join the org if that would interest you.

**Pull request merged**